pkg/rpc/rpc.proto

```diff
@@ -1,7 +1,11 @@
 syntax="proto3";
 
+package longhorn.io.longhorn.instance.manager.pkg.rpc;
```

Comment:
Same as the backing image manager PR.
I prefer `longhorn.instancemanager.pkg.rpc`

Reply:
Sounds good.